src/rustls.h

```diff
@@ -1740,6 +1751,56 @@ rustls_result rustls_client_config_builder_set_key_log(struct rustls_client_conf
                                                        rustls_keylog_log_callback log_cb,
                                                        rustls_keylog_will_log_callback will_log_cb);
 
+/**
+ * Configure the client for Encrypted Client Hello (ECH).
+ *
+ * This requires providing a DER encoded list of ECH configurations that should
```

Comment:
reminder to self: these aren't DER. Need to do a consistency pass. It's a TLS encoded structure.

Reply:
fixed.